### PR TITLE
as3-select-methods: done

### PR DESCRIPTION
**Pull request by ghbooth12**

### CODE REVIEW

File: lib/bloc_record/validation.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module Validation
```
Comment (ghbooth12) [inline lib/bloc_record/validation.rb]: Bloc Assignment
>1. Add input validation, errors, and error handling to BlocRecord.

File: lib/bloc_record/selection.rb [hunk 2]
```diff
@@ -1,6 +1,12 @@
 require 'sqlite3'
 
 module Selection
+  def method_missing(m, *args, &block)
```
Comment (robmadden) [inline lib/bloc_record/selection.rb]: Gahee, you mentioned in our last conversation that you weren't sure about this working correctly. From the instructions:

> If you implement this correctly, the commands Entry.find_by(:name, name) and Entry.find_by_name(name) will have identical results.

In other words, you can write a spec that does this exactly which would verify the functionality for yourself.

Comment (robmadden) [inline lib/bloc_record/selection.rb:4]: After further review, I believe you implemented this correctly.

Comment (robmadden) [reply lib/bloc_record/selection.rb:4]: The instructions are not super clear here, but your implementation appears to be correct (at least for the one case you have, `find_by_name`.) Sorry for any confusion.

**Pull request closed without merging**